tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -522,3 +522,33 @@ Feature: collaboration (wopi)
       | app-endpoint                                              |
       | /app/open-with-web?file_id=<<FILEID>>&app_name=FakeOffice |
       | /app/open-with-web?file_id=<<FILEID>>                     |
+
+
+  Scenario Outline: open file with non-existing file id (open-with-web)
```

Comment:
this test is about sending post requests on deleted file rather than non existent. So either rewrite that here or use fake file id generate from function

Reply:
file-id of deleted file also doesn't exist in ocis.

Reply:
but how would you like to call the scenario should not matter in this case.

Reply:
in surface it looks same but in-depth its different. If deleted file id still works then how do we distinguish whats the real problem file id not deleted or random file id giving response

Reply:
Scenario Outline: tries to open file with deleted file id (open-with-web)